Rakefile

```diff
@@ -4,6 +4,7 @@ Rake::TestTask.new(:test) do |t|
   t.libs << "test"
   t.libs << "lib"
   t.test_files = FileList["test/**/*_test.rb"]
+  t.warning = false
```

Comment:
Why disabling warnings?

Reply:
Because there are too many of them - hiding actual "weird" messages, and in the case of Ruby, warnings are not *that* critical (i.e. for me a clear case of "disable them until you took the time to fix all of them").

Reply:
I would tend to forget fixing them, but ok